StripeFinancialConnections/StripeFinancialConnections/Source/Helpers/FinancialConnectionsFont.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  FinancialConnectionsFont.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 5/2/23.
+//
+
+import Foundation
+import UIKit
+
+struct FinancialConnectionsFont {
+
+    let uiFont: UIFont
+    let lineHeight: CGFloat
+
+    var topPadding: CGFloat {
+        return ((lineHeight - uiFont.lineHeight) / 2) + (uiFont.ascender - uiFont.capHeight)
```

Comment:
Will this be still valid lineHeight is smaller than uiFont's lineHeight? Padding might end up being negative? Maybe we do some abs()?

Reply:
good catch, thank you, I will max the `(lineHeight - uiFont.lineHeight) / 2)` to 0